Makefile

```diff
@@ -75,6 +75,7 @@ docker-push:
 e2e-setup:
 	ginkgo version
 	ginkgo -v -r e2e/setup
+	kubectl wait --for=condition=Ready pod -l app.kubernetes.io/name=vela-core -n vela-system --timeout=300s
```

Comment:
@wonderflow @ryanzhang-oss  To make sure oam-controller pod is running before other e2e-test start, I employ this approach from oam-kubernetes-runtime project. Otherwise, tests related to interacting with oam-controller may fail or timeout.

Reply:
Maybe `vela-core` we have changed this name?

Reply:
currently, the value of `app.kubernetes.io/name` is still "vela-core", and `app.kubernetes.io/instance` is "kubevela"
let make it more rigorous as below:
`kubectl wait --for=condition=Ready pod -l app.kubernetes.io/name=vela-core,app.kubernetes.io/instance=kubevela -n vela-system --timeout=300s`